code/modules/antagonists/_common/antag_team.dm

```diff
@@ -167,8 +167,11 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 	if(!message)
 		return
 
+	var/team_message = chat_box_red("<font color='#d6000b'><span class='bold'>Admin '[name]' Team Message ([user.key]): </span></font><span class='notice'>[message]</span>")
+	var/team_alert_sound = sound('sound/effects/adminticketopen.ogg')
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, team_alert_sound)
+		to_chat(M.current, "[team_message]")
```

Comment:
This is already a text message, isnt it? No need to put it in a string

Reply:
I blame Sean ... somehow